### ISSUE TITLE
Bug in example 16, duplicated keyword to save residuals file

### ISSUE THREAD

**Issue by rapoliveira** [rpoleski/MulensModel#111]
The current version of the file [ob03235_2_full.yaml](https://github.com/rpoleski/MulensModel/blob/master/examples/example_16/ob03235_2_full.yaml) generates the following error in example 16:
```
Traceback (most recent call last):
  File "ulens_model_fit.py", line 3234, in <module>
    ulens_model_fit.run_fit()
  File "ulens_model_fit.py", line 564, in run_fit
    self._parse_other_output_parameters()
  File "ulens_model_fit.py", line 900, in _parse_other_output_parameters
    raise ValueError('Unrecognized key: ' + str(key) + "\n " +
ValueError: Unrecognized key: other_output
 Expected keys: models
```
It happens because the key `other_output` is nested inside the same key in line 108, when giving the information for the residuals file.

Removing this duplication and leaving `models`, `yaml output` and `residuals` as keys of `other_output` solves the problem.

**Comment by rapoliveira** [comment]
@rpoleski, I addressed this minor issue and added 1.0 to the second Y axis, as you suggested.
Please let me know if you have more ideas to implement here in example16.

**Comment by rapoliveira** [comment]
Solved in pull request #114, I will close the issue.

**Comment by rapoliveira** [comment]
@rpoleski can the pull request #114 be reopened, so that I add today changes to second Y-axis?
Actually I edited only [2 lines](https://github.com/rapoliveira/MulensModel/commit/90dbfab5b919499ba10fd5752c71410bd1aa59a7).
